docs/UserGuide.md

```diff
@@ -310,156 +310,156 @@ the constraints of each parameter when used in a command.
 
 Adds a patient and their relevant details to MediBase3.
 
-Format: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
+**Format**: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
 > 
 > * All fields are compulsory and must be non-empty.
-> * A patient will not be added if the NRIC given is already associated with another patient in MediBase3. An error message will be displayed in this case.
+> * If the `NRIC` provided already exists in MediBase3, an error message will display, and the patient will not be added.
 > * The new patient will be added to the end of the Patient List Panel.
 > * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
 
-Examples:
+**Examples**:
 * `add n/John Doe i/S1234567A g/M d/2002-12-12 p/98765432 e/johnd@example.com a/Orchard Road, Block 124, #02-01`
 * `add n/Betsy Crowe i/s1234567b g/F e/betsycrowe@example.com a/Bukit Merah, Block 123, #01-01 p/1234567 d/2002-11-10`
 
 {: .alert .alert-success}
 > :bulb: **Tip:**
 > 
 > * Remember that `NRIC` and `GENDER` are case-insensitive.
-> E.g. `i/s1234567a` and `i/S1234567A` are both equivalent.
+>   -  e.g. `i/s1234567a` and `i/S1234567A` are both equivalent.
 > * Made a mistake or a typo? You can use the [`edit` command](#editing-a-patient--edit) to update the patient's details.
 
 [Back to Table of Contents](#table-of-contents)
 
 #### Deleting a patient : `delete`
 
-Deletes the specified patient and their details from MediBase3.
+Removes a patient and their details from MediBase3.
 
-Format: `delete NRIC`
+**Format**: `delete NRIC`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
 > 
-> * Deletes the patient with the specified `NRIC` in MediBase3.
-> * The `NRIC` provided must be the **full NRIC** of the patient to be deleted. e.g. `S1234567A` and not `S123`.
-> * You can delete a patient even if they are not being currently displayed in the Patient List Panel.
-> * `NRIC` must adhere to the constraints mentioned in the [Parameter Details](#Parameter-Details) section.
+> * Deletes the patient with the specified `NRIC` from MediBase3.
+> * The `NRIC` provided must be complete, 
+>   - e.g. `S1234567A` (not partial, like `S123`).
```

Comment:
Might be better to make it consistent,
![image](https://github.com/user-attachments/assets/7ad006b1-230d-44a4-bd05-01c335d8dcd1)

Stick to what you wrote for `edit`